bin/elasticity-flexture-L0/1_visco_elastic_deformation.ipynb

```diff
@@ -0,0 +1,112 @@
+{
```

Comment:
<p>Suggested rephrasing of last sentence: "....&nbsp;how long it takes for stresses in a deformed viscoelastic system to decay to a steady state"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geodynamics-hubzero/gem-notebooks/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>